### PR TITLE
added readMe file

### PR DESCRIPTION
**Pull request by TarasYurkovskyi**

### CODE REVIEW

File: readme.md
Comment (d1sam) [inline readme.md:1]: please add possible improvements section.

Comment (d1sam) [reply readme.md:1]: Optionally, you can create a collection of your Postman requests, attach it, and explain how users can utilize it.

**Review by d1sam — CHANGES_REQUESTED** [review]
good job, but there are some changes should be done!

File: pom.xml [hunk 1]
```diff
@@ -117,10 +117,6 @@
             <artifactId>jjwt-jackson</artifactId>
             <version>${jjwt.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-docker-compose</artifactId>
-        </dependency>
```
Comment (yevheniisynytsia) [inline pom.xml:123]: why did you remove it?

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
++ `PATCH: /api/orders/{id}` - endpoint for updating orders status (Admin)
++ `GET: /api/orders/{orderId}/items` - endpoint for getting order items from order (User)
++ `GET: /api/orders/{orderId}/items/{itemId}` - endpoint for getting specific item from order (User)
++ `GET: /api/cart` - endpoint for getting all items into shopping cart (User)
++ `POST: /api/cart` - endpoint for adding items in shopping cart (User)
++ `PUT: /api/cart/cart-items/{itemId}` - endpoint for updating items quantity (User)
++ `DELETE: /api/cart/cart-items/{itemId}` - endpoint for deleting items from shopping cart (User)
+## How to run the project from your side?
+* Start by making a local copy of the project's repository
+* Create the .env file with the corresponding variables
```
Comment (yevheniisynytsia) [inline readme.md:67]: it is not trivial
can you please add a template for it? or specify the list of variables that should be present in the `.env` file?

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
Looks good, but I left one important thing to consider

File: env.sample [hunk 3]
```diff
@@ -0,0 +1,8 @@
+MYSQL_USER=your_mysql_user
+MYSQL_ROOT_PASSWORD=your_mysql_root_password
+MYSQL_DATABASE=your_mysql_database
+MYSQL_LOCAL_PORT=your_local_mysql_port
+MYSQL_DOCKER_PORT=your_docker_mysql_port
+SPRING_LOCAL_PORT=your_local_spring_port
+SPRING_DOCKER_PORT=your_docker_spring_port
+DEBUG_PORT=your_debug_port
```
Comment (d1sam) [inline env.sample]: please add one empty line at the end of file.

File: pom.xml (hunk 1 above)
Comment (d1sam) [reply pom.xml:123]: still exists.

**Review by d1sam — CHANGES_REQUESTED** [review]
there are still some minor changes that should be done!

File: pom.xml (hunk 1 above)
Comment (TarasYurkovskyi) [reply pom.xml:123]: i have some problems with ci build so i decided to remove this dependency thinking it will help. But problem was not here and i forgot to add it back and push, in the last commit i add it back)

**Review by d1sam — APPROVED** [review]
well done!
